doc/rear.8.adoc

```diff
@@ -578,19 +577,36 @@ _/usr/share/doc_.
 
 
 == AUTHORS
-The ReaR project was initiated in 2006 by Schlomo Schapiro (https://github.com/schlomo) and Gratien D'haese (https://github.com/gdha) and has since then seen a lot of contributions by many authors. As ReaR deals with bare metal disaster recovery, there is a large amount of code that was contributed by owners and users of specialized hardware and software. Without their combined efforts and contributions ReaR would not be the universal Linux bare metal disaster recovery solution that it is today.
-
-As time passed the project was lucky to get the support of additional developers to also help as maintainers: Dag Wieers (https://github.com/dagwieers), Jeroen Hoekx (https://github.com/jhoekx), Johannes Meixner (https://github.com/jsmeix), Vladimir Gozora (https://github.com/gozora) and Sébastien Chabrolles (https://github.com/schabrolles). We hope that ReaR continues to prove useful and to attract more developers who agree to be maintainers.  Please refer to the MAINTAINERS (https://github.com/rear/rear/blob/master/MAINTAINERS) file for the list of active and past maintainers.
-
-To see the full list of authors and their contributions please look at the git history (https://github.com/rear/rear/graphs/contributors). We are very thankful to all authors and encourage anybody interested to take a look at our source code and to contribute what you find important.
+The ReaR project was initiated in 2006 by Schlomo Schapiro (https://github.com/schlomo) and
+Gratien D'haese (https://github.com/gdha) and has since then seen a lot of contributions by many authors.
+As ReaR deals with bare metal disaster recovery, there is a large amount of code
+that was contributed by owners and users of specialized hardware and software.
+Without their combined efforts and contributions ReaR would not be
+the universal Linux bare metal disaster recovery solution that it is today.
+
+As time passed the project was lucky to get the support of additional developers to also help as maintainers:
+Dag Wieers (https://github.com/dagwieers),
+Jeroen Hoekx (https://github.com/jhoekx),
+Johannes Meixner (https://github.com/jsmeix),
+Vladimir Gozora (https://github.com/gozora),
+Sebastien Chabrolles (https://github.com/schabrolles),
+Renaud Metrich (https://github.com/rmetrich) and
+"pcahyna" (https://github.com/pcahyna).
```

Comment:
Pavel Cahyna

Reply:
@pcahyna 
I didn't write your real name there
because you don't show it on
https://github.com/pcahyna
so I assumed you prefer to keep your real name private.

Reply:
Done via https://github.com/rear/rear/pull/2819/commits/8f40d02fa8717659426b2e3ce02ecea71202dc4b